source/material_model/reactive_fluid_transport.cc

```diff
@@ -0,0 +1,353 @@
+                             "from the base model. Units: dimensionless.");
+          prm.declare_entry ("Reference fluid viscosity", "10",
+                             Patterns::Double (0),
+                             "The value of the constant melt/fluid viscosity $\\eta_f$. Units: \\si{\\pascal\\second}.");
+          prm.declare_entry ("Exponential fluid weakening factor", "27",
+                             Patterns::Double (0),
+                             "The porosity dependence of the viscosity. Units: dimensionless.");
+          prm.declare_entry ("Reference permeability", "1e-8",
+                             Patterns::Double(),
+                             "Reference permeability of the solid host rock."
+                             "Units: \\si{\\meter\\squared}.");
+          prm.declare_entry ("Fluid compressibility", "0.0",
+                             Patterns::Double (0),
+                             "The value of the compressibility of the fluid. "
+                             "Units: \\si{\\per\\pascal}.");
+          prm.declare_entry ("Fluid reaction time scale for operator splitting", "1e3",
+                             Patterns::Double (0),
+                             "In case the operator splitting scheme is used, the porosity field can not "
+                             "be set to a new equilibrium fluid fraction instantly, but the model has to "
+                             "provide a reaction time scale instead. This time scale defines how fast fluid "
+                             "release and absorption happen, or more specifically, the parameter defines the "
+                             "time after which the deviation of the porosity from the free fluid fraction "
+                             "that would be in equilibrium with the solid will be reduced to a fraction of "
+                             "$1/e$. So if the fluid reaction time scale is small compared "
+                             "to the time step size, the reaction will be so fast that the porosity is very "
+                             "close to this equilibrium value after reactions are computed. Conversely, "
+                             "if the fluid reaction time scale is large compared to the time step size, almost no "
+                             "fluid release and absorption will occur."
+                             "\n\n"
+                             "Also note that the fluid reaction time scale has to be larger than or equal to the reaction "
+                             "time step used in the operator splitting scheme, otherwise reactions can not be "
+                             "computed. If the model does not use operator splitting, this parameter is not used. "
+                             "Units: yr or s, depending on the ``Use years "
+                             "in output instead of seconds'' parameter.");
+          prm.declare_entry ("Fluid solid reaction scheme", "no reaction",
```

Comment:
I think I would prefer this with a dash
```suggestion
          prm.declare_entry ("Fluid-solid reaction scheme", "no reaction",
```